feat: Update SSO and Internal LB (#336)
CluedIn-io/CluedIn.Documentation: docs/020-deployment/howtos/020-setup-sso.md

1. Under **Manage**, select **App registrations** > **New registration**.
1. Enter a display **Name** for your application.
![Register_application_Name.png](../../assets/images/ama/howtos/configure-sso-2.png)
1. Select the **Supported account types** that can use the application.

> 1. Leave Redirect URI empty for the time being.

Reviewer: ```suggestion
1. Leave **Redirect URI** empty for the time being.
```

1. Select **Register**.

>     When the registration finishes, the Azure portal displays the **Overview** pane of the application registration. Here you can see the **Application (client) ID**. This value uniquely identifies your application. Make note of this Id as you'll need it to [enable SSO via Helm](#create-kubernetes-secret-and-enable-sso-via-helm).

Reviewer: ```suggestion
    When the registration finishes, the Azure portal displays the **Overview** pane of the application registration. Here you can see the **Application (client) ID**. This value uniquely identifies your application. Make note of this ID as you'll need it to [enable SSO via Helm](#create-kubernetes-secret-and-enable-sso-via-helm).
```

**To expose the API**

1. In the Azure portal, in **App registrations**, select your application.
1. Select **Expose an API**.
1. In the **Scopes defined by this API** section, select **Add a scope**.

> 1. For the first time setup, you will need to specify the `Application ID URI`. For this, use your frontend url with `/sso` appended. (ie. `https://cluedin.yourdomain.com/sso`)

Reviewer: ```suggestion
1. For the first-time setup, you will need to specify the `Application ID URI`. For this, use your front-end URL with `/sso` appended. (i.e. `https://cluedin.yourdomain.com/sso`)
```

    ![configure-sso-scopename.png](../../assets/images/ama/howtos/configure-sso-scopename.png)

> 1. Specify the following scope attributes and then click `Add scope`:

Reviewer: ```suggestion
1. Specify the following scope attributes and then select `Add scope`:
```
